src/test/java/org/opentripplanner/routing/fares/OrcaFareServiceTest.java

```diff
@@ -91,6 +91,17 @@ public void calculateFareWithNoFreeTransfer() {
         calculateFare(rides, Fare.FareType.electronicYouth, 0f + DEFAULT_RIDE_PRICE_IN_CENTS + 175f);
     }
 
+    @Test void calculateFareByLeg() {
```

Comment:
Add a method comment to explain what the test is proving and move the `void calculateFareByLeg()` to a new line.